app/src/test/kotlin/co/netguru/baby/monitor/client/feature/batterylevel/NotifyLowBatteryUseCaseTest.kt

```diff
@@ -23,9 +24,9 @@ class NotifyLowBatteryUseCaseTest {
         notifyLowBatteryUseCase.notifyLowBattery(title, text)
 
         verify(notificationSender).broadcastNotificationToFcm(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/7c069c26782df05dac8a097aa956e351ad95f920/app/src/test/kotlin/co/netguru/baby/monitor/client/feature/batterylevel/NotifyLowBatteryUseCaseTest.kt#L26"></span>Unexpected indentation (11) (it should be 12)</td>
    </tr>
  </tbody>
</table>